change/@ni-nimble-components-450fdd90-7444-494f-8c1f-8e7aba8c7804.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I noticed some inconsistencies when testing on macOS:
1. In Firefox there is still no link preview on drag. However if you drop the non-existent/invisible preview then it does actually navigate.
2. in Safari and Chromium the drag preview for most links shows a title in addition to a URL but the nimble anchor only shows a URL. 

I'm not sure of the cause but one hint is that if I replace the content of the `<a>` in the shadow DOM with text instead of spans then I do see that text as the link preview in FF and the title in Safari/Chrome. 

## Firefox

https://github.com/ni/nimble/assets/10500124/83ad50ec-3d66-44a1-8578-e671165f7033

## Safari

https://github.com/ni/nimble/assets/10500124/da1bcd56-2a6d-46a1-9b1e-b3a0bc3fe55e

Reply:
From what I can tell, both Chrome and Firefox are not showing the slotted content as part of their drag preview. The difference between Chrome and Firefox is that Chrome also shows the href in addition to the anchor content, which is why in Chrome the preview isn't completely blank.

I played around with custom drag previews a little bit to see how viable that solution would be, and I haven't yet gotten to something that works correctly.

I'll have to play around with it some more, but I'm not sure how much custom logic we want to try to add.

Reply:
Looks like this bug has already been filed to Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1589364

The bad news is that it was filed in 2019 and hasn't gotten a lot of traction.

Reply:
I'd be ok with proceeding with the current approach and just waiting for browsers to eventually fix the bugs. Workarounds like a custom drag preview seem low ROI and possibly a burden to maintain / test.

It'd be good to make sure Chromium has a similar bug.

Reply:
I couldn't find the same bug in Chromium, which makes sense because the overall drag behavior of custom elements is significantly better in Chrome than in FF. I created a new bug specific to the anchor drag behavior: https://issues.chromium.org/issues/329489154